### PR TITLE
[Download] Release Swift 5.9.1

### PR DESCRIPTION
**Pull request by shahmishal**
Screenshot 2023-10-18 at 6 15 12 PM

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test

**Review by alexandersandberg — APPROVED** [review]

**Review by cthielen — APPROVED** [review]

File: download/_build-platform.html [hunk 1]
```diff
@@ -66,6 +66,3 @@
         {% endfor %}
     </tbody>
 </table>
-
-<sup>1</sup> Swift {{ include.platform.name }} is available as part of <a href="https://itunes.apple.com/app/xcode/id497799835">{{ include.platform.xcode }}</a>.<br>
```
Comment (stevapple) [inline download/_build-platform.html:70]: Why is this line removed?

File: download/_build-platform.html [hunk 2]
```diff
@@ -66,6 +66,3 @@
...
-<sup>2</sup> Swift {{ include.platform.name }} Windows 10 toolchain is provided by <a href="https://github.com/compnerd">Saleem Abdulrasool</a>. Saleem is the platform champion for the Windows port of Swift and this is an official build from the Swift project. <br><br>
```
Comment (stevapple) [inline download/_build-platform.html:71]: We can add this one conditionally according to Swift version. I don’t think it should be dropped for existing releases.

File: download/_build-platform.html [hunk 3]
```diff
@@ -52,7 +50,9 @@
                 {% endif %}
                 {% assign tag_downcase = include.platform.tag | downcase %}
                 <a href="https://download.swift.org/{{ tag_downcase }}/{{ platform_name_url }}/{{ include.platform.tag}}/{{ include.platform.tag}}-{{ platform_name }}.{{ package_extension }}">{{ arch }}</a>
+                {% if platform.name != "Windows 10" %}
```
Comment (stevapple) [inline download/_build-platform.html:53]: We’re still documenting how to verify the signature of the Windows toolchain, which works pretty well (and is depended by some CI actions IFAIK). Can you expand on why we’re hiding this for Windows?

File: _data/builds/swift_releases.yml [hunk 4]
```diff
@@ -1325,3 +1325,40 @@
...
+      archs:
+        - x86_64
+        - aarch64
+    - name: Red Hat Universal Base Image 9
+      docker: 5.9.1-rhel-ubi9
+      dir: ubi9
+      archs:
+        - x86_64
+        - aarch64
+    - name: Windows 10
```
Comment (stevapple) [inline _data/builds/swift_releases.yml:1362]: Since Microsoft has released Windows 11 a pretty long while ago, and it’s binary compatible with Windows 10, it makes more sense to change this into "Windows 10/11", even if it’s `windows10` internally. Adding 11 here would make it more intuitive for new users.

**Review by stevapple — COMMENTED** [review]
Sorry for being late in this thread. I noticed a few regressions from the "Download" page after 5.9.1 release (most of which are Windows-related), so I may want to ask here first.

File: _data/builds/swift_releases.yml (hunk 4 above)
Comment (shahmishal) [reply _data/builds/swift_releases.yml:1362]: @compnerd

File: download/_build-platform.html (hunk 3 above)
Comment (shahmishal) [reply download/_build-platform.html:53]: I will get this updated

File: download/_build-platform.html (hunk 2 above)
Comment (shahmishal) [reply download/_build-platform.html:71]: good point, we should add Swift 5.9.0 or older check.

File: download/_build-platform.html (hunk 1 above)
Comment (shahmishal) [reply download/_build-platform.html:70]: We need to fix this statement, as it's not true with all of the releases.

File: download/_build-platform.html (hunk 2 above)
Comment (stevapple) [reply download/_build-platform.html:71]: Implemented in #429
